src/executor/trt_graph_executor.cc

```diff
@@ -434,7 +435,7 @@ Executor *TrtGraphExecutor::TensorRTBind(nnvm::Symbol symbol,
                                          std::unordered_map<std::string, NDArray> *shared_buffer,
                                          Executor *shared_exec) {
   auto exec = new exec::TrtGraphExecutor();
-  exec->Init(symbol, default_ctx, group2ctx,
+  exec->Init(std::move(symbol), default_ctx, group2ctx,
```

Comment:
Is Init accepting an rvalue reference?

Reply:
It's a good question I wanted to dive into a little.  My understanding is that the Init function can bind its variable to either an lvalue or an rvalue when passing by value. You don't have to create an overload or anything to determine which one it expects.  Here's an example: 

https://gist.github.com/KellenSunderland/d431119580a116410c672b9535d85170
Which is using code borrowed from: 
https://www.chromium.org/rvalue-references?tmpl=%2Fsystem%2Fapp%2Ftemplates%2Fprint%2F&showPrintDialog=1

Reply:
Well the question if the move is doing anything, because I don't know if there's an Init signature accepting an rvalue reference to a symbol, that was the original question.